TCAT/RouteDetailSection.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  RouteDetailSection.swift
+//  TCAT
+//
+//  Created by HAIYING WENG on 11/6/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+enum RouteDetailSection{
```

Comment:
Add space after section {

Reply:
Also, here you create a special section enum for our route details table. I am aware that Omar has another enum for "Section" - How do we feel about potentially renaming this to also be more specific? @Omarrasheed

Reply:
So I think I had mentioned this to Haiying earlier (not 100% sure) but I think we actually want to move off of having that global `Section` enum and rather create private enums within each of the viewControllers that have their tableviews. The reason for this is because we don't always use some of the enum types, like in the homeOptionsCard we don't ever have to show a `currentLocation` and thus we just fall through with that case in the switch. This should not be the case, as we should only have enum cases that we actually use. Lets keep this for now and I'll spin up an issue to make this refactor cc @kevinchan159 @danielVebman

Reply:
totally agree. was not aware we structured it like this. btw a useful read if you haven't read this already:
https://bible.cornellappdev.com/standards/ios/tableviews